config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     olm.skipRange: '>=4.0.0 <5.0.0'
```

Comment:
@alinaryan , consider squashing the commits, no need for two separated commits.

Reply:
squashed